kimtaeyoung/5월/week02/백준_1427번문제.py

```diff
@@ -0,0 +1,18 @@
+# 소트인사이드
+
+''' 문제
+배열을 정렬하는 것은 쉽다. 수가 주어지면, 그 수의 각 자리수를 내림차순으로 정렬해보자.
+'''
+
+''' 입력
+첫째 줄에 정렬하려고 하는 수 N이 주어진다. N은 1,000,000,000보다 작거나 같은 자연수이다.
+'''
+
+''' 출력
+첫째 줄에 자리수를 내림차순으로 정렬한 수를 출력한다.
+'''
+
+# 풀이
+a = list(input())
+a.sort(reverse=True)
+print(*a, sep='')
```

Comment:
```suggestion
print(''.join(a))
	```
리스트 괄호없이 출력하는 방법이 이렇게 간단한지 몰랐네요!
찾아보니 join함수를 이용한 방법도 있네요!